manim/animation/creation.py

```diff
@@ -1,4 +1,40 @@
-"""Animate the display or removal of a mobject from a scene."""
+r"""Animate the display or removal of a mobject from a scene.
+
+.. manim:: CreationModule
+    :hide_source:
```

Comment:
why hide the source code?

Reply:
This is shown at the top of the module, so I thought it'd be nice to use this rendered scene as sort of a banner or teaser for the module and its contents. I kinda want to show at a glance what this module is about. The code would be important to show how to recreate this scene, but given that the scene is so niche, I'm not sure that would be helpful to anyone...

Reply:
Okay, I see, good idea!